core/chainio/avs_subscriber.go

```diff
@@ -358,6 +363,19 @@ func (s *AvsSubscriber) getLatestTaskFromEthereumV2() (*servicemanager.ContractA
 	// The second topic is the batch merkle root, as it is an indexed variable in the contract
 	latestTask.BatchMerkleRoot = lastLog.Topics[1]
 
+	// return the task if has not been responded only
```

Comment:
This functionality currently is useful, but the function name does not indicate this.
This functionality should also refactor its name to `getLatestNonRespondedTask` or similar, to indicate it.

Reply:
Also, unrelated, I understand you inspired your getTask using the query from this function. But it is wrong and there is an issue about this incorrect implementation: https://github.com/yetanotherco/aligned_layer/issues/978